### ISSUE TITLE
Growth Curve Table Set  (for manuscripts)

### ISSUE THREAD

**Issue by wibeasley** [IALSA/IALSA-2015-Portland#130]
(*Spun off from Issue #97*)

Now that the plan for the correlation tables have stabilized (issue #129). @ampiccinin, @andkov, and @wibeasley concentrated today's meeting on the "growth curve table set".

@ampiccinin tentative first choice resembles the screenshot below.  

* Each table is dedicated to a specific measure; each column is a specific to a study.
* Each cell is actually an average of multiple estimates.  For instance, the fictitious `0.4688` in the top left cell is the average estimate of the bnt when it was paired with all ~4 physical processes.
* The tables will have a different number of columns, because not all studies have all measurements.
* It will actually be double the number of columns b/c the variables are regressed on each other (*I didn't quite understand, but I do think it need to yet.*)

![image](https://cloud.githubusercontent.com/assets/1372890/16053890/43091dd8-3230-11e6-9ef8-208bb52ddb4b.png)

* The next step is for @andkov to re-introduce these estimates into the parser.  He's creating an issue for this.
* @andkov & @wibeasley will then produce this version, and another version where the columns represent the betas (instead of representing the studies).  Hopefully it won't be too disorienting to users accustomed to the initial version.  There are a few programming advantages to it.  Also, it might be easier to compare estimates between studies (say, *b1*) when you're scanning up/down a column, instead of left/right across a row.

**Comment by andkov** [comment]
@wibeasley , the issue #132  has implemented the expansion of the parser to include covariate estimates. The  updated [catalog](https://github.com/IALSA/IALSA-2015-Portland/blob/master/data/shared/parsed-results.rds) now includes all six covariates. Please try for a draft of the table sketched above.

**Comment by andkov** [comment]
Sketch of the table system:

![fullsizerender 3](https://cloud.githubusercontent.com/assets/2608073/16437190/66f74b96-3d59-11e6-97f7-9a273d992418.jpg)

**Comment by andkov** [comment]
The generic table in which row = process and col = various contexts in which this process was statistically related (for process bnt: grip-bnt, gait-bnt, fev-bnt, etc)

|study|model_type|subgroup|process_type|process_name| mean estimate|context 1|...|context K|
|---|---|---|---|---|---|---|---|---|
|elsa|aehplus|female|grip|bnt|3.50| 3.00|...|4.00|

Each column/context represents the measure, with with the row measure was statistically related during estimation (gait, grip, fev, animals, bnt, etc + univar)

**Comment by andkov** [comment]
## Questions to address
- there will be twice as many rows in orange and yellow, does that make a difference in our plans?
- the range of values across various statistical contexts is quite substantial (especially for the intercept of the second level), how do we deal with it?

**Comment by wibeasley** [comment]
During the call with @ampiccinin, @GracielaMuniz, and @andkov, we decided that the growth curve table will show up only in the study-specific papers --and not in the meta paper.  If the meta readers want this information, they can go to the dynamic tables hosted on the workshop's repo.

The study-specific papers should resemble this one below.  These sections will exist for the level2 intercept, and then repeated immediately below for the level 2 slope.  The column/process order should resemble the grouping of the [outcome space map](https://raw.githubusercontent.com/IALSA/IALSA-2015-Portland/master/reports/outcome-space/figures_rmd/outcome-space-map-1.png).

@ampiccinin & @GracielaMuniz, please add respond this explanation isn't consistent with your plans, or if there's another detail I left out.

![image](https://cloud.githubusercontent.com/assets/1372890/16664334/22558f50-4445-11e6-8ff9-10a2420011fb.png)

**Comment by ampiccinin** [comment]
@andkov  - how easy would it be to create a version of the "seed" tables that included "process A" only as a single column next to the Process B (cognitive) output? We could select the process A version to include by just picking the one that comes first alphabetically (or some other arbitrary method, that is equally implementable in each study). We can't realistically include all of the different process A models in the appendix, it is difficult to see how any of the relatively small differences matter, it takes a bunch of time to reformat each of those tables for publication (in an appendix) for each study and each paper, and the information will be available for the curious and obsessive on GitHub. It was important to be able to look at them ourselves, but it is too much to provide to the typical reader.

Assuming it will be much faster for you to programming an alternate reformatted version of the table (kind of like the full and focussed versions of the correlations) than for multiple people to reformat each table for each study and each project by hand.

Thanks!!!

**Comment by wibeasley** [comment]
> but it is too much to provide to the typical reader

Word.

@andkov, I'm getting my projects mixed up.  For this one, have we considered placing a 'master copy' (that's too big for normal consumption w/o filtering) in a [DT](https://rstudio.github.io/DT/), that's on a GitHub.io page?  It should be easier to reference from a static/paper article than a knitr report that's located on the conventional side of the repository.

**Comment by andkov** [comment]
@ampiccinin, Good point, simpler is better. Let me think about what you have said, how to implement it. Would you have 10-15 mins to chat tomorrow to help me understand?

**Comment by andkov** [comment]
@wibeasley, no we haven't spoken about this. I was considering turning it into a GitHub.io website, so this might be a good time.

So you think to host just the DT? You think it might be too big to be imbedded in a report to host through rawgit?

**Comment by ampiccinin** [comment]
I'll be heading to Vancouver for the day, but since you are 3 hours ahead, we could try before I go, or maybe  while I wait for the ferry. Or now?

**Comment by andkov** [comment]
@ampiccinin, Can't now, sorry, too sleepy. I'm usually up by 5 am local, so pretty much any time your morning works for me. Give me some warning, if possible, and call my cell any time. If you can point me to the report, we don't really need a screenshare.

**Comment by ampiccinin** [comment]
Will do. 

the table/report to which I refer is, for example:
[seed-elsa 2017-02-07.docx](https://github.com/IALSA/IALSA-2015-Portland/files/953659/seed-elsa.2017-02-07.docx)

So - for example - you would insert the "word_de" column for the rows next to process "a" down next to the process "b" variables (I would suggest in first column after "label"). The whole top sections of the table (all rows "ab" and "a" would then drop out. Then, down in the variances section, you would just have to print the relevant ones "a" for "a" and "b" for "b". Finally, the last bit could remain as-is.

I hope this makes sense when you look at the file! If you send questions in the morning, I can think about the answer before I call.

**Comment by andkov** [comment]
@ampiccinin, 

![image](https://cloud.githubusercontent.com/assets/2608073/25530734/500665bc-2bf5-11e7-8983-6b39141d84a6.png)

> create a version of the "seed" tables that included "process A" only as a single column next to the Process B (cognitive) output? 

> for example,  insert the "word_de" column for the rows next to process "a" down next to the process "b" variables (I would suggest in first column after "label"). The whole top sections of the table (all rows "ab" and "a" would then drop out. Then, down in the variances section, you would just have to print the relevant ones "a" for "a" and "b" for "b". Finally, the last bit could remain as-is.

**Comment by andkov** [comment]
@ampiccinin , so you would like to drop some rows entirely and add a column that contains info for one process?
![image](https://cloud.githubusercontent.com/assets/2608073/25530964/1cb9a5ba-2bf6-11e7-941b-c9e7df35f8cf.png)

**Comment by andkov** [comment]
![image](https://cloud.githubusercontent.com/assets/2608073/25532300/ee6d3262-2bfa-11e7-9824-20b7855c6ac5.png)

**Comment by ampiccinin** [comment]
@andkov  - I will call. Here is what I was trying to describe! :)

[ABcombine example.docx](https://github.com/IALSA/IALSA-2015-Portland/files/965213/ABcombine.example.docx)
